components/prism-editorconfig.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.editorconfig = {
+	// https://editorconfig-specification.readthedocs.io/en/latest/
+	'comment': /[;#].*$/m,
+	'section': {
+		pattern: /^[ \t]*\[.+]/m,
+		alias: 'keyword',
+		inside: {
+			'regex': /\\\\[\[\]{},!?.*]/, // Escape special characters with '\\'
+			'punctuation': /[\[\]{},!?]|\.\.|\*{1,2}/
+		}
+	},
+	'property': /^[ \t]*[^\s=]+?(?=[ \t]*=)/m,
```

Comment:
The `[^\s=]+?` doesn't have to be lazy because we know that the character after it will be either a space, tab, or equals sign because of the lookahead.

```suggestion
	'property': /^[ \t]*[^\s=]+(?=[ \t]*=)/m,
```